SPECS/tidy/tidy.spec

```diff
@@ -91,13 +92,39 @@ rm -fv %{buildroot}%{_libdir}/libtidys.a
 %{_libdir}/pkgconfig/tidy.pc
 
 %changelog
+* Tue Oct 18 2022 Osama Esmail <osamaesmail@microsoft.com> - 5.8.0-5
+- Upgraded from 5.7.28 to 5.8.0
+- libtidys.a -> libtidy.a
+- Added make
```

Comment:
Please be more descriptive with these two lines, because they don't say much to anyone reading it, who doesn't have the full context of the changes you're doing right now.